subsys/net/ip/net_context.c

```diff
@@ -82,6 +82,32 @@ static enum net_verdict packet_received(struct net_conn *conn,
 static void set_appdata_values(struct net_pkt *pkt, enum net_ip_protocol proto);
 
 #if defined(CONFIG_NET_TCP)
+static int send_reset(struct net_context *context, struct sockaddr *remote);
+
+static struct tcp_backlog_t {
+	struct net_tcp *tcp;
+	struct sockaddr remote;
+	u32_t recv_max_ack;
+	u32_t send_seq;
+	u32_t send_ack;
+	bool cancelled;
```

Comment:
It might be better to put the bool as a last in the struct as it is probably the shortest one.